BrcmPatchRAM/BlueToolFixup.cpp

```diff
@@ -64,6 +64,20 @@ static const uint8_t kVendorCheckPatched[] =
     0xEB // jmp short
 };
 
+static const uint8_t kVendorCallbackOriginal[] =
```

Comment:
Without context this is very vague imo. I suggest a comment with a brief explanation + link to PR

Reply:
Done